www/searchutil.php

```diff
@@ -257,6 +257,7 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
         $selectList = "distinct games.id as id,
                        games.title as title,
                        games.author as author,
+                       games.desc as description,  // On the home page, descriptions are shown for recommended games
```

Comment:
`//` comments don't work inside quotation marks. This breaks the search.

Reply:
Oops. I can take it out.

Reply:
Done